src/hotspot/share/opto/phasetype.hpp

```diff
@@ -168,6 +169,8 @@ class PhaseNameValidator {
         // strncpy always writes len characters. If the source string is shorter, the function fills the remaining bytes with NULLs.
         strncpy(_bad, *iter, len);
         _valid = false;
+      } else if (PHASE_ALL == cpt) {
+        mask = ~((uint64_t)0);
```

Comment:
Can we also change this from uint64_t to UINT64_C for the consistent style?

Reply:
done